_posts/2021-04-22-azure-solutions-architect-networking.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "[Notatka] Architektura w Azure - Networking"
+categories:
+- "Architecture Meetings"
+tags:
+- "Virtual Networks"
+- "Subnets"
+- "VPN Gateway"
+author:
+- aryll
+- maciejgos
+---
+
+## Notatka ze spotkania 'Architektura w Azure - Networking' #azuresolutionsarchitect
+
+{% youtube JvOYNWkMCcA %}
+
+### O czym dyskutowaliśmy:
+- O skali projektów - kiedy pojawają się zagadnienia związane z infrastrukturą sieciową
+- Jak infrastrukturą wygląda w projektach - w jaki sposób jest dzielona na środowiska dev, oraz prod?
+- O subnet-ach - należy pamiętać, że kazda podsieci traci na starcie 5 adresów które są wykorzystywane przez Azure
+- VPN Gateway vs VPN Gateway - kiedy tego użyć oraz co to jest?
+- Network Security Group vs Application Security Group - kiedy użyć oraz co to jest?
+- Virtual WAN - czym jest i jakie problemy rozwiązuje
+- Czym różni się połączenie Site-To-Site i Point-to-Site?
+
+
+### Linki
+- [Architektury Azure](https://docs.microsoft.com/en-us/azure/architecture/browse/)
```

Comment:
to obiecałam że dorzucimy do linków
https://www.youtube.com/watch?v=7EJtmX1Vcmc

Reply:
done